pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -928,6 +928,14 @@ self: super: {
   # Work around overspecified constraint on github ==0.18.
   github-backup = doJailbreak super.github-backup;
 
+  # https://github.com/andrewthad/chronos/issues/62
+  # doctests are failing on newer GHC versions
+  chronos = unmarkBroken (dontCheck super.chronos);
+
+  # The chronos dependency is what's causing this to fail, due to the above
+  # issue with doctests
+  co-log = unmarkBroken super.co-log;
```

Comment:
Instead of marking these unbroken here, can you just remove them from the `broken-packages` list in `configuration-hackage2nix.yaml`?

Here is a PR doing something similar: https://github.com/NixOS/nixpkgs/pull/108788/files#diff-3139f03f40f0c7740a4003f88522ac0af603af77f746f3e6bb8e26cccc1eaf88L8334

This this, you can remove the `co-log = ` line, and just keep the `chronos = dontCheck super.chronos;` line.

Also, thanks for linking to the upstream issue for `chronos`!